src/system/ddt.70

```diff
@@ -66,6 +66,14 @@ setf [Type of disk? (0 none, 1 Systems Concepts, 2 RP-10, 3 RH-10)]dsksw
 	define rh10
 	ife dsksw-3!termin
 
+ifdef rh10,[
```

Comment:
I see dsktp= indeed should only be asked if rh10 is set.  So how come it shows up in the ks10 build?

Reply:
I'll enter a separate ticket on this.